mm_classification/server.py

```diff
@@ -113,8 +113,10 @@ def classify(array, classifier_id):
 
     # return array of 1's and 0's
     return output
-  
 
-if __name__ == '__main__':
+# callable function to run server, included this in order to define our poetry entrypoint
+def main():
     app.run(debug=False, port=5001, host='0.0.0.0')
```

Comment:
Is this necessarily correct?  When invoking, this indicates running on several interfaces!